pymeasure/instruments/thyracont/smartline_v2.py

```diff
@@ -0,0 +1,505 @@
+
+    def __init__(self, parent, id):
+        if id is None:
+            id = self._id
+        assert id == self._id, f"Pirani ID has to be {self._id}."
+        super().__init__(parent, id=self._id)
+
+    pressure = Channel.measurement(
+        "0M{ch}00", """Get the current pressure in mbar.""",
+        preprocess_reply=lambda r: r.replace("UR", "0").replace("OR", "inf"),
+    )
+
+
+class Pirani(SensorChannel):
+    """Pirani sensor channel.
+
+    A Pirani sensor measures the heat transfer in the gas. The reading depends on the gas type.
+    The sensor reading is linear to the real pressure below some threshold, around 1 mbar.
+    You may control a gas factor with :attr:`gas_factor`.
+    """
+
+    _id = 1
+
+    gas_factor = gas_factor
+
+    statistics = Channel.measurement(
+        "0PM011",
+        """Get the sensor statistics: wear in percent (negative: corrosion,
+            positive: contamination), time since last adjustment in hours.""",
+        preprocess_reply=lambda msg: msg.strip("W"),
+        separator="A",
+        cast=int,
+        get_process=lambda vals: (vals[0], vals[1] / 4),
+    )
+
+
+class Piezo(SensorChannel):
+    """Piezo sensor channel. A piezo sensor is independent of the gas present."""
+
+    _id = 2
+
+
+class HotCathode(SensorChannel):
+    """Hot cathode sensor channel."""
+
+    _id = 3
+
+    filament_mode = Instrument.control(
+        "0FC00", "2FC01%i",
+        docs="""Control which hot cathode filament to use.
+        ("2 if 1 defect", "Filament1", "Filament2", "toggle>1mbar")
+        """,
+        values={"2 if 1 defect": 0,
+                "Filament1": 1,
+                "Filament2": 2,
+                "toggle>1mbar": 3},
+        validator=validators.strict_discrete_set,
+        map_values=True,
+        cast=int,
+        check_set_errors=True,
+    )
+
+    degas = Instrument.control(
+        "0DG00", "2DG01%i",
+        """Control the degas mode.""",
+        values={True: 1, False: 0},
+        map_values=True,
+        validator=validators.strict_discrete_set,
+        check_set_errors=True,
+    )
+
+    sensor_enabled = Instrument.control(
+        "0CC00", "2CC01%i",
+        """Control the state of the cathode.""",
+        values={True: 1, False: 0},
+        map_values=True,
+        validator=validators.strict_discrete_set,
+        check_set_errors=True,
+    )
+    active_filament = Instrument.measurement(
+        "0FN00", "Get the current filament number.",
+        cast=int,
+    )
+
+    filament_status = Instrument.measurement(
+        "0FS00", """Get the status of the hot cathode filaments.""",
+        values=["Filament 1 and 2 ok",
+                "Filament 1 defective",
+                "Filament 2 defective",
+                "Filament 1 and 2 defective"],
+        map_values=True,
+    )
+
+    gas_factor = gas_factor
+
+    statistics = Channel.measurement(
+        "0PM013",
+        """Get the wear in percent: of filament 1, of filament 2.""",
```

Comment:
```suggestion
        """Get the filaments wear level in percent. Return a tuple of two values (wear of filament 1,  wear of filament 2).""",
```
I am not sure whether this is better, just to make sure I understood your docstring. If correct, feel free to leave your comment as it is or change it

Reply:
I improved it (also the other statistics docstring)